morl_baselines/multi_policy/gpi_pd/gpi_pd.py

```diff
@@ -873,7 +877,7 @@ def train(
                     n_value = policy_evaluation_mo(self, eval_env, wcw, rep=num_eval_episodes_for_front)[3]
                     linear_support.add_solution(n_value, wcw)
 
-            if self.log:
+            if self.log and self.global_step % (eval_freq * 10) == 0:
```

Comment:
The idea here is to log multipolicy metrics after the end of every iteration. Why this change?

Reply:
Because the evaluation is super slow in practice and I don't think we lose that much info when evaluating less in this case. I mean, it took 5 days to run GPI-LS for 3M steps IIRC 🫠

Reply:
I see, but this depends on the number of timesteps per iteration too, and the hardcoded 10 is misleading. I think this should be split into two parameters:

eval_freq: Inside an iteration, how often in time steps to do policy evaluation
x (idk a good name) : Eval the multi-policy metrics at every x iterations, in the outer loop.

Reply:
I initially had `eval_freq_in_iter` and `eval_freq` to do that, which I merged through the *10 :p. But yeah, we can add a parameter.

Reply:
I added an additional parameter `eval_mo_freq`